bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/CcuGateway.java

```diff
@@ -149,7 +150,13 @@ protected void addChannelDatapoints(HmChannel channel, HmParamsetType paramsetTy
 
     @Override
     protected void setVariable(HmDatapoint dp, Object value) throws IOException {
-        String strValue = Objects.toString(value, "").replace("\"", "\\\"");
+        // If Double then anders umwandeln
```

Comment:
Half German? :) Since readable from the code below that it's converted differently for double, I would propose simply dropping the comment.

Reply:
Oh f... I simply forgot to remove this note to myself :-(.